src/main/java/com/sericulture/authentication/model/entity/UserInfo.java

```diff
@@ -40,8 +42,9 @@ public class UserInfo {
     @Column(name = "email_id")
     private String emailID;
 
-    @Column(name = "role_id")
-    private Long roleId;
+    @OneToOne(cascade = CascadeType.ALL)
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'member def modifier' has incorrect indentation level 4, expected level should be 2.